src/infill.cpp

```diff
@@ -181,6 +181,10 @@ void Infill::addLineInfill(Polygons& result, const PointMatrix& rotation_matrix,
     int scanline_idx = 0;
     for(int64_t x = scanline_min_idx * line_distance + shift; x < boundary.max.X; x += line_distance)
     {
+        if (scanline_idx >= cut_list.size())
```

Comment:
I still think you should handle this case differently.

If this can sometimes occur we should change `line_count` on infill.cpp:302 and make it one larger or something.

(See earlier discussion at https://github.com/Ultimaker/CuraEngine/commit/e4803ea6af3910fc5de1c8e50fb18ed1cbe6c19e#commitcomment-22851737)